### DISCUSSION TITLE
Button label not in between

### DISCUSSION THREAD

**Discussion by mohitxskull** [mantinedev/mantine#6402] [Q&A]
![image](https://github.com/mantinedev/mantine/assets/101568337/27c107cd-4db9-421c-afcb-bc6f5ce39471)

**Comment by atareversei** [comment]
Can you provide a sandbox?

**Reply by mohitxskull** [reply]
Heyy @atareversei sorry forgot to reply you back!

i will share a sandbox with you, for now we added top margin on button text
